### DISCUSSION TITLE
How to pin all of LazyVim plugins

### DISCUSSION THREAD

**Discussion by kachidk** [LazyVim/LazyVim#1222] [Q&A] [answered]
I want to pin all default LazyVim plugins (plugins that come with LazyVim)

 how do I do it in the best way possible?

Or perhaps this could be a new feature 

I want to ensure stability for up to a year

**Comment by dpetka2001** [comment, marked as answer]
Pinning the plugins means they won't get updated. You might miss on new features or any BREAKING changes by doing that. I wouldn't suggest that. If you update your Neovim, something might break as well with the plugins not being updated. Except if you also don't update your Neovim for up to a year. Are you sure this is what you want? Also other plugins might break as well, except if you're only using the default ones that come preinstalled. Lazyvim itself might break because it's a plugin itself. I would suggest, if stability is your end goal, to have a regular backup of your dotfiles and preferably in a version controlled manner as well (for example github). And also take notice that `Lazy` itself provides the user with an option to revert to an older commit, if you experience some kind of breakage.
